types/src/v17/blockchain/mod.rs

```diff
@@ -650,6 +650,22 @@ pub struct PruneBlockchain(
     pub i64,
 );
 
+/// Result of JSON-RPC method `verifychain`.
+///
+/// > verifychain
+/// >
+/// > Arguments:
+/// > 1. checklevel (numeric, optional, default=3, range=0-4) How thorough the block verification is:
+/// - level 0 reads the blocks from disk
+/// - level 1 verifies block validity
+/// - level 2 verifies undo data
+/// - level 3 checks disconnection of tip blocks
+/// - level 4 tries to reconnect the blocks
+/// - each level includes the checks of the previous levels
+/// > 2. nblocks (numeric, optional, default=6, 0=all) The number of blocks to check.
```

Comment:
```suggestion
/// Result of JSON-RPC method `verifychain`.
```

I've not been perfectly uniform but lets try to not include docs for optional args. Just because its too noisy and even more of a problem with going stale between versions.

So for these 'thin' methods perhaps just the one line rustdoc is good enough?

Reply:
True, let me do this...